recipe-client-addon/lib/ClientEnvironment.jsm

```diff
@@ -174,6 +179,21 @@ this.ClientEnvironment = {
       return Preferences.get("privacy.donottrackheader.enabled", false);
     });
 
+    XPCOMUtils.defineLazyGetter(environment, "experiments", async () => {
```

Comment:
These lazy getters run once on first use, and then get cached forever, yeah? What happens when we run every 24 hours, do these lists never update? Maybe these should be normal getters, instead of lazy getters?